lib/core/merge-webpack-config.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./dist/server/utils/merge-webpack-config').default;
```

Comment:
what is this for ?

Reply:
Good catch! I added an entry-point for that to give people access to the merge function, but I should remove it. Doing that now.